resources/views/layouts/master.blade.php

```diff
@@ -10,15 +10,15 @@
 	<title>CuapCuap &middot; @yield('title')</title>
 
 	@section('external-resource')
-		<link rel="stylesheet" href="http://cdn.everything.io/kickstart/3.x/css/kickstart.min.css" />
-		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+			<link rel="stylesheet" href="{!! asset('css/kickstart.min.css) !!}">
```

Comment:
asset() doesn't redirect to /resources/assets because /resources/assets is for storing raw files. 
Move /resources/assets/css to /public/css